### ISSUE TITLE
Do we need new extra-scidb-libs for scidb 19.11.3?

### ISSUE THREAD

**Issue by ksens** [Paradigm4/extra-scidb-libs#29]
@dgosselin-p4 @rvernica 

@dgosselin-p4 mentioned that we need updated extra-scidb-libs for working with 19.11.3 (what is the ticket reference for this?) 

As noted in an email sent by me:

> As soon as I started using scidb 19.11.3 (extra-scidb-libs is at 19.11-1), I started seeing random scidb crashes like below:

```sh
Apr 12 06:12:04 6812842adfc0 shim[37]: execute_query[s6tb2y]: execute, scidb[0] 0x7f168c042e10, scidb[1] 0x7f168c042e40, query create array public.FEATURE_ENTITY_FLEX_FIELDS 
< feature_metadata_value_id : int64 > [ entity_id=0:*,2,0, featureset_id=0:*,32,0, metadata_attrkey_id=0:*,2048,0, entity_base_id=0:*,65536,0 ]
Apr 12 06:12:04 6812842adfc0 shim[37]: execute_query[s6tb2y]: execute, qid 0.1586671924566180100
Apr 12 06:12:04 6812842adfc0 shim[37]: execute_query: ERROR execute, s6tb2y: SystemException in file: src/network/BaseConnection.h function: receive line: 168
                                       Error id: scidb::SCIDB_SE_NETWORK::SCIDB_LE_CANT_SEND_RECEIVE
                                       Error description: Network error. Cannot receive network message: Read failed: End of file (asio.misc:2).  An instance may be offline..
```

> On another try, the same query ran OK.

If I use a combination of scidb 19.11.2 / 19.11.0 and extra-scidb-libs 19.11-1, I see no similar errors. 

This was on SciDB CE. I have not upgraded any of my EE machines to scidb 19.11.3 yet. 

We are tracking a bug at a customer https://paradigm4.atlassian.net/browse/SDB-6871 that has scidb 19.11.3 and extra-scidb-libs 19.11-1. Might be unrelated but thought I would check here.

**Comment by JAKinchen** [comment]
I'll upgrade the extra-scidb-libs package and install it at PDQ.  Don't know if it will solve this problem, but it should be done regardless.

**Comment by JAKinchen** [comment]
Check that - I'll wait until Dave says it's okay.

**Comment by ksens** [comment]
This seems to be resolved with the release of `extra-scidb-libs-19.11-2`.
